lab-matt/index.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
This works!  Though you could instead create a single module which performs the reading task with a specified file to read instead of the way you've solved the issue here.  The way you have it now with the first, second, and third functions works, but doesn't follow DRY methodology since you're repeating yourself.